src/neuroconv/datainterfaces/ecephys/openephys/openephyslegacydatainterface.py

```diff
@@ -74,6 +74,14 @@ def __init__(
             es_key=es_key,
         )
 
+        if ignore_timestamps_errors and self.recording_extractor.has_time_vector():
+            for segment_index in range(self.recording_extractor.get_num_segments()):
+                self.recording_extractor._recording_segments[segment_index].time_vector = None
+            warn(
+                "The recording has a time vector but the timestamps might be discontinuous. "
+                "The time vector was reset to None to avoid writing these timestamps with the electrical series."
+            )
+
```

Comment:
Intuitively seems perfectly fine - only thing to check is if the logic on the SI side relies on _value_ of the `.time_vector is None` rather than on the logic `hasattr(recording_segment, "time_vector")`

Reply:
No matter what we have time to polish this; this shouldn't be merged to `main` until both neo and SI get new releases with the upstream fixes

Tye lab repo will have to pin to either this branch (to stay up-to-date with other things updated from main) or a commit from this branch